mmv1/templates/terraform/examples/network_security_security_profile_group_basic.tf.erb

```diff
@@ -0,0 +1,19 @@
+resource "google_network_security_security_profile_group" "<%= ctx[:primary_resource_id] %>" {
+  provider                  = google-beta
+  name                      = "my-security-profile-group"
```

Comment:
This should use a ctx variable so that it gets a `tf-test` prefix and a random suffix - see https://googlecloudplatform.github.io/magic-modules/develop/test/test/#add-a-create-test

Reply:
done